lib/api/src/grpc/validate.rs

```diff
@@ -116,6 +116,42 @@ impl Validate for crate::grpc::qdrant::quantization_config_diff::Quantization {
     }
 }
 
+impl Validate for crate::grpc::qdrant::condition::ConditionOneOf {
+    fn validate(&self) -> Result<(), ValidationErrors> {
+        use crate::grpc::qdrant::condition::ConditionOneOf;
+        match self {
+            ConditionOneOf::Field(field_condition) => field_condition.validate(),
+            ConditionOneOf::Nested(nested) => nested.validate(),
+            ConditionOneOf::Filter(filter) => filter.validate(),
+            ConditionOneOf::IsEmpty(_) => Ok(()),
+            ConditionOneOf::HasId(_) => Ok(()),
+            ConditionOneOf::IsNull(_) => Ok(()),
+        }
+    }
+}
+
+impl Validate for crate::grpc::qdrant::FieldCondition {
+    fn validate(&self) -> Result<(), ValidationErrors> {
+        // TODO && self.geo_polygon.is_none()
```

Comment:
this must be added once https://github.com/qdrant/qdrant/pull/2315 lands
@zzzz-vincent

Reply:
ack. Thanks for letting me know.